test/Thesis/Composition/Helpers.hs

```diff
@@ -0,0 +1,16 @@
+module Thesis.Composition.Helpers where
+
+import Thesis.Types
+import Thesis.Composition.PrivacyFilter
+import Thesis.ValueGuard
+
+mkPrice :: Epsilon -> Delta -> QueryPrice
+mkPrice epsilon delta =
+  case (positive epsilon, nonNegative delta) of
+    (Right epsilon, Right delta) -> (epsilon, delta)
```

Comment:
This gives a warning about pattern matches being non-exhaustive. I have been using the `error` function for now, but we need a long term solution.

Reply:
I've turned off the warning for non-exhaustive patterns in tests.

Generally, I think that both solutions are acceptable in tests. They can't lead to a runtime error in production, and they serve only as a helper to alleviate writing too much code.